app/controllers/application_templates_controller.rb

```diff
@@ -23,6 +23,20 @@ def toggle_active
     redirect_to :back
   end
 
+  def toggle_eeo_enabled
+    @template.toggle! :eeo_enabled
+    if @template.eeo_enabled
+      show_message :eeo_enabled,
+                   default:
+      'You have enabled EEO data requests on your application.'
```

Comment:
I think you could probably pare this down to something shorter (and more meaningful) like

```
EEO data will [not] be requested for this application.
```

which would mean you can avoid splitting a key/value pair with a newline.